src/dlls/mscoree/unixinterface.cpp

```diff
@@ -238,8 +243,21 @@ int coreclr_initialize(
     {
         host.SuppressRelease();
         *hostHandle = host;
-    }
+#ifdef FEATURE_GDBJIT
 
+        hr = coreclr_create_delegate(*hostHandle,
+                                     *domainId,
+                                     "System.Diagnostics.Debug.SymbolReader",
+                                     "System.Diagnostics.Debug.SymbolReader.SymbolReader",
+                                     "GetInfoForMethod",
+                                     (void**)&getInfoForMethodDelegate);
+
+        if (!SUCCEEDED(hr))
+        {
+            fprintf(stderr, "coreclr_create_delegate failed - status: 0x%08x\n", hr);
```

Comment:
Could you please change the error message to be more specific? For example by adding the full name of the function (`System.Diagnostics.Debug.SymbolReader.SymbolReader.GetInfoForMethod`) to the message.